### PR TITLE
[READY FOR REVIEW]Fixes readme badges

### PR DESCRIPTION
**Pull request by Gideon357**
Fixes the badges in the readme. Originally in PR #1487

### CODE REVIEW

**Comment by Gideon357** [comment]
@jacobtomlinson I added the GH actions badges, but they offer a number of "event types." I added the one for the push event (CI), though that might be incorrect. I think the default badge shows all pull requests, and the push only shows pushes. In any case, I think this should be merged.

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "MD033" : false,
+    "MD013": false
+}
```
Comment (jacobtomlinson) [inline .markdownlint.json:4]: Looks like you're running markdown lint here and have some config leaking into your PR.

We don't currently lint our markdown, maybe we should. Either way this should be removed from this PR. 

Maybe we should open an issue about linting markdown.

File: README.md [hunk 2]
```diff
@@ -6,14 +6,12 @@
 
 <p align=center>
 <a href="https://pypi.python.org/pypi"><img src="https://img.shields.io/pypi/v/opsdroid.svg" alt="Current version of pypi" /></a>
-<a href="https://travis-ci.org/opsdroid/opsdroid"><img src="https://img.shields.io/travis/opsdroid/opsdroid/master.svg?logo=travis" alt="Build Status" /></a>
-<a href="https://ci.appveyor.com/project/opsdroid/opsdroid/branch/master"><img src="https://img.shields.io/appveyor/ci/opsdroid/opsdroid/master.svg?logo=appveyor" alt="Build status" /></a>
+<a href="https://github.com/opsdroid/opsdroid/actions"><img src="https://github.com/opsdroid/opsdroid/workflows/CI/badge.svg?event=push" alt="Github CI Status"></img></a>
```
Comment (jacobtomlinson) [inline README.md:9]: I think we want to specifically have this showing the master branch.

File: README.md [hunk 3]
```diff
@@ -6,14 +6,12 @@
...
 <a href="https://bettercodehub.com/"><img src="https://bettercodehub.com/edge/badge/opsdroid/opsdroid?branch=master" alt="BCH compliance" /></a>
 <a href="https://pyup.io/repos/github/opsdroid/opsdroid/"><img src="https://pyup.io/repos/github/opsdroid/opsdroid/shield.svg" alt="Updates" /></a>
 <a href="https://hub.docker.com/r/opsdroid/opsdroid/builds/"><img src="https://img.shields.io/docker/build/opsdroid/opsdroid.svg" alt="Docker Build" /></a>
-<a href="https://hub.docker.com/r/opsdroid/opsdroid/"><img src="https://img.shields.io/microbadger/image-size/opsdroid/opsdroid.svg" alt="Docker Image" /></a>
-<a href="https://microbadger.com/#/images/opsdroid/opsdroid"><img src="https://img.shields.io/microbadger/layers/opsdroid/opsdroid.svg" alt="Docker Layers" /></a>
+<img alt="Docker Image Size (latest by date)" src="https://img.shields.io/docker/image-size/opsdroid/opsdroid?label=image%20size"><a href="https://microbadger.com/#/images/opsdroid/opsdroid"><img src="https://img.shields.io/microbadger/layers/opsdroid/opsdroid.svg" alt="Docker Layers" /></a>
```
Comment (jacobtomlinson) [inline README.md:14]: Looke like the link around the immage size badge has been lost here.

File: .markdownlint.json (hunk 1 above)
Comment (Gideon357) [reply .markdownlint.json:4]: My bad! Sorry!

**Comment by Gideon357** [comment]
> Should we remove the appveyor badge as well? It's showing `project not found or access denied` and we have ditched appveyor right? 🤔

@FabioRosado I belive I did that, but I will make sure

**Comment by Gideon357** [comment]
@jacobtomlinson is there a reason that we do the badges in HTML? It seems simpler to do them in Markdown.

**Comment by jacobtomlinson** [comment]
It is simpler but you have less control. The main reason was to get things centrally aligned. See #906 for info.

**Comment by Gideon357** [comment]
@jacobtomlinson if the checks all pass, this should be ready to merge. Thanks for your help!

File: README.md (hunk 2 above)
Comment (jacobtomlinson) [reply README.md:9]: I would still like this to explicitly be set to master. 

```
![CI](https://github.com/opsdroid/opsdroid/workflows/CI/badge.svg?branch=master&event=push)
```

Comment (jacobtomlinson) [reply README.md:9]: And the link should go here https://github.com/opsdroid/opsdroid/actions?query=workflow%3ACI

**Comment by jacobtomlinson** [comment]
Thanks for bearing with me on this!

**Comment by Gideon357** [comment]
@jacobtomlinson  thank you so much for you help!

**Pull request merged**